ml-agents-envs/mlagents/envs/subprocess_env_manager.py

```diff
@@ -90,6 +95,11 @@ def _send_response(cmd_name, payload):
                 _send_response("reset", all_brain_info)
             elif cmd.name == "global_done":
                 _send_response("global_done", env.global_done)
+            elif cmd.name == "timers":
```

Comment:
@harperj Let's talk about this - I'm not sure if it's better to add a separate command to request the timers, or return them in `step` response. The latter is probably easier with async stepping, but the downside is that it might make it difficult to time `_send_response()`.

Reply:
(not going to fix the breaking unit test until we decide what to do)